assets/js/src/licences.js

```diff
@@ -0,0 +1,25 @@
+const licenseSelect = document.getElementById('licencesspdxID');
+$(document).ready(function() {
+  getLicenses();
+});
+
+function getLicenses() {
+  $.getJSON(
+    'https://raw.githubusercontent.com/spdx/license-list-data/master/json/licenses.json',
```

Comment:
I'm doing a little research to see if we could just do this during the Jekyll build, since pulling it form githubuserconten on each page load might not be the best approach